test/jest/utils/antelope/globalProps.ts

```diff
@@ -1,31 +0,0 @@
-
-// -------- globalProps for vue --------
-
-const globalProps = {
-    $t: jest.fn(() => 'Mocked translation'),
-    $warningNotification: jest.fn(),
-};
-
-jest.mock('vue', () => {
-    const originalVue = jest.requireActual('vue');
-
-    return {
-        ...originalVue,
-        getCurrentInstance: jest.fn(() => ({
-            appContext: {
-                config: {
-                    globalProperties: globalProps,
-                },
-            },
-        })),
-        ComponentInternalInstance: originalVue.ComponentInternalInstance,
-        computed: originalVue.computed,
-        defineComponent: originalVue.defineComponent,
-        watch: originalVue.watch,
-    };
-});
-
-export {
-    globalProps,
-};
-
```

Comment:
vue dependency should not be overwritten. any changes required on vue instances should be made in `jest.init.js`